gitbook/audio-spatialization.md

```diff
@@ -2,91 +2,91 @@
 
 ## Introduction
 
-By default, sounds and music are played at full volume in each speaker; they are not *spatialized*. 
+By default, sounds and music are played at full volume in each speaker; they are not *spatialized*.
 
-If a sound is emitted by an entity which is to the right of the screen, you would probably want to hear it from the right speaker. If a music is being played behind the player, you would want to hear it from the rear speakers of your Dolby 5.1 sound system. 
+If a sound is emitted by an entity which is to the right of the screen, you would probably want to hear it from the right speaker. If a music is being played behind the player, you would want to hear it from the rear speakers of your Dolby 5.1 sound system.
 
-How can this be achieved? 
+How can this be achieved?
 
 ## Spatialized sounds are mono
 
-A sound can be spatialized only if it has a single channel, i.e. if it's a mono sound.  
-Spatialization is disabled for sounds with more channels, since they already explicitly decide how to use the speakers. This is very important to keep in mind. 
+A sound can be spatialized only if it has a single channel, i.e. if it's a mono sound.
+Spatialization is disabled for sounds with more channels, since they already explicitly decide how to use the speakers. This is very important to keep in mind.
 
 ## The listener
 
-All the sounds and music in your audio environment will be heard by a single actor: the *listener*. What is output from your speakers is determined by what the listener hears. 
+All the sounds and music in your audio environment will be heard by a single actor: the *listener*. What is output from your speakers is determined by what the listener hears.
 
-The class which defines the listener's properties is [Listener]({{book.api}}/Listener.html). Since the listener is unique in the environment, this class only contains static functions and is not meant to be instantiated. 
+The class which defines the listener's properties is [Listener]({{book.api}}/Listener.html). Since the listener is unique in the environment, this class only contains static functions and is not meant to be instantiated.
 
-First, you can set the listener's position in the scene: 
+First, you can set the listener's position in the scene:
 
-```
-sf::Listener::setPosition(10.f, 0.f, 5.f);
+```ruby
+SF::Listener.set_position(SF.vector3(10.0, 0.0, 5.0))
```

Comment:
Class methods have the suffix, it's expected.

`Vector3f` is not like `Vector2`, doesn't have this nice alternative.

What works is `SF::Vector3f.new(x:0,y:0,z:0)`, but I will make `SF.vector3f(0, 0, 0)` and this is what should be used here.